functions/supplier/mail-supplier/README.adoc

```diff
@@ -25,10 +25,12 @@ All configuration properties are prefixed with `mail.supplier`.
 
 For more information on the various options available, please see link:src/main/java/org/springframework/cloud/fn/supplier/mail/MailSupplierProperties.java[MailSupplierProperties].
 
+A `ComponentCustomizer<MailInboundChannelAdapterSpec<?, ?>>` (or `ComponentCustomizer<ImapIdleChannelAdapterSpec>` when `mail.supplier.idle-imap = true`) bean can be added in the target project to provide any custom options for the `MailInboundChannelAdapterSpec` (or `ImapIdleChannelAdapterSpec`, respectively) configuration used by the `mailSupplier`.
```

Comment:
Nice way to handle both in single sentence, the `respectively` in the 2nd or helps and it reads very clean.